ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/priv/PrivGetLogs.java

```diff
@@ -85,9 +100,17 @@ public JsonRpcResponse response(final JsonRpcRequestContext requestContext) {
         requestContext.getRequest().getId(), new LogsResult(matchingLogs));
   }
 
+  private List<LogWithMetadata> getEmptyList() {
```

Comment:
Do we need this extra method? Why not return `Collections.emptyList()` straight away on line 78?

Reply:
compile error!